addons/mail/static/src/models/messaging_menu/messaging_menu.js

```diff
@@ -49,17 +49,7 @@ function factory(dependencies) {
             if (!this.isOpen) {
                 return;
             }
-            const inbox = this.env.messaging.inbox;
-            if (
-                !inbox ||
-                !inbox.mainCache ||
-                inbox.mainCache.isLoaded ||
-                inbox.mainCache.isLoading
-            ) {
-                return;
-            }
-            // populate some needaction messages on threads.
-            inbox.mainCache.update({ hasToLoadMessages: true });
+            this.env.messaging.inbox.mainCache.update({ isCacheRefreshRequested: true });
```

Comment:
I don't like how users of caches now have to trigger load messages. Setting `isCacheRefreshRequested: true` is super cryptic in my opinion, and I really dislike that. It's not clear what a cache refresh is, and the "requested" part makes it even more obscure because it suggests that cache can do anything with this field, at least just ignore it. This is alike a method call that does nothing because the method shouldn't have been called in the 1st place.

Same remark as other comment: if PR fixes the behaviour is 13.4 and we don't have to change anything in this version (at worse tiny changes), then it's bearable for 13.4... but clean this ASAP in master.